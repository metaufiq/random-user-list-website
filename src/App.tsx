import React from 'react';
import UserList from './pages/UserList';

function App() {
  return <UserList/>;
}

export default App;
