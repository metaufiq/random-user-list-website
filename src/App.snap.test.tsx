import { assertSnapshots } from "../test";
import App from "./App";

jest
  .mock('./pages/UserList', () => 'UserList')

describe('AppSnapshot', () => {

  const configs = [
    {
      desc: 'should render normal App'
    }
  ];

  assertSnapshots(App, configs);
});
