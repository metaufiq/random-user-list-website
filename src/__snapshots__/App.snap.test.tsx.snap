// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSnapshot should render normal App 1`] = `<UserList />`;
